test/extended/cmd.sh

```diff
@@ -74,75 +75,74 @@ popd
 rm -rf $tmp
 
 # ensure a local-only image gets a docker image(not imagestream) reference created.
-[ "$(oc new-app test/scratchimage~https://github.com/openshift/ruby-hello-world.git --strategy=docker -o yaml |& tr '\n' ' ' | grep -E "from:\s+kind:\s+DockerImage\s+name:\s+test/scratchimage:latest\s+")" ]
+current_project="$( oc project -q )"
+os::cmd::expect_success 'oc new-project test-scratchimage'
+os::cmd::expect_success 'oc new-app test/scratchimage~https://github.com/openshift/ruby-hello-world.git --strategy=docker'
+os::cmd::expect_success_and_text 'oc describe bc' 'From Image:\s+DockerImage test/scratchimage:latest'
```

Comment:
@deads2k instead of some convoluted `jsonpath`, *etc*, I decided it would be much easier to reason about what was actually being tested if we broke it up like this.